src/utils/LogsUtil.ts

```diff
@@ -62,8 +62,8 @@ export function getAllLogObjects(): Array<Log> {
     const exists = checkLogsJson();
     if (exists) {
         const filepath = getLogsJson();
-        const rawLogs = fs.readFileSync(filepath).toString();
-        return JSON.parse(rawLogs).logs;
+        const rawLogs = getFileDataAsJson(filepath);
```

Comment:
like these you can call "getFileDataAsJson(filepath, {logs: []);". but up to you , we can have it handled they way you have it now.